src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -244,7 +247,9 @@ applyGenesisCmd logger dbEnv spv cmd =
         , _txGasModel = _geGasModel freeGasEnv
         }
 
-    interp = initStateInterpreter $ initCapabilities [magic_GENESIS, magic_COINBASE]
+    interp = initStateInterpreter
+      $ disableNatives nativesToDisable
```

Comment:
For now I'd just make one function/definition `disablePact40Natives` to not crowd the namespace with this one change.